extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -468,7 +471,12 @@ function _mapDisplaySets(
                   switch (action.id) {
                     case 'yes':
                       try {
-                        await dataSource.reject.series(ds.StudyInstanceUID, ds.SeriesInstanceUID);
+                        if (dataSource.reject) {
```

Comment:
I am a bit surprised that the "client" logic landed outside of the data sources themselves. Users of a data source should be shielded from such changes. We should consider leaving the code here as it was before and having the logic for choosing a client reject in the client manager/data source.